install/kubernetes/helm/subcharts/gateways/templates/clusterrole.yaml

```diff
@@ -15,6 +15,25 @@ rules:
   resources: ["virtualservices", "destinationrules", "gateways"]
   verbs: ["get", "watch", "list", "update"]
 ---
+{{- if eq $key "istio-ingressgateway" }}
+{{- if .Values.ingressGatewayAgent.enabled }}
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ $key }}-agent-{{ .Release.Namespace }}
+  namespace: istio-ingress
+  labels:
```

Comment:
For new PRs - please don't add this. We are planning to use other installers, and label changes are tricky on upgrade. 
We need to keep them where they're already defined.

Reply:
Removed. Thanks for letting me know.